### ISSUE TITLE
Menu items require 2 clicks on iOS 13 (tested on iPad Pro 12.9 2018)

### ISSUE THREAD

**Issue by lsascha** [foundation/foundation-sites#11865]
## What should happen?
Clicking on menu items should redirect to the link right away
...

## What happens instead?
Having to click 2 times to open a menu item.
...

## Possible Solution

...

## Test Case and/or Steps to Reproduce (for bugs)
Same behaviour can be observed on the foundation docs site.
https://foundation.zurb.com/sites/docs/

## Your Environment

- Foundation version(s) used: 6.3.1, but can as mentioned be obersved on the foundation docs site too.
- Browser(s) name and version(s): Safari ? (the one included in iOS 13?), Chrome v77.0.3865.69
- Device, Operating System and version: iPad Pro 12.9 2018, iOS 13
- Link to your project: https://www.wolterskluwer.de or as mentiond i could reproduce it on https://foundation.zurb.com/sites/docs/ too.

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.

**Comment by DanielRuf** [comment]
Hi @lsascha,

there are some options at https://foundation.zurb.com/sites/docs/dropdown-menu.html#js-options.

Can you check if this also happens on iOS 12 and before and also on other devices?

**Comment by SassNinja** [comment]
@lsascha imo it's working as expected!

what is supposed to happen on your example page if the ipad user clicks on `Newsroom`?
immediate execution of the href without any chance to click the opened children (e.g. `News & Presse`) in your dropdown menu?

I can only imagine a new option to disable all children for specific breakpoints (if you wanna avoid duplicate markup)

**Comment by DanielRuf** [comment]
> @lsascha imo it's working as expected!

Right, At least this is how it should be for a menu with children to open it on hover / click / tap on mobile. This is not a bug in this way.

**Comment by Sharrnah** [comment]
Problem does not happen with iOS 12.
Also i am not talking about menu items with children, but items that are just a href links to different pages. These should be clickable and open on the first click.

**Comment by lsascha** [comment]
Sorry.
Problem does not happen on iOS12,

and its about menu items that are direct links to different pages,
not talking about menu items with children here.

**Comment by DanielRuf** [comment]
So it's a Safari regression / bug and not caused by changes on our side.

**Comment by rafibomb** [comment]
It seems to be related to hover. Setting `data-disable-hover="true"` will solve for iPad but then you cant hover open the desktop version.

**Comment by joeworkman** [comment]
This issue has been mentioned on **Foundation Open Source Community**. There might be relevant details there:

https://foundation.discourse.group/t/ipad-menu-click-issues/912/11

**Comment by wfehr** [comment]
Having the same problem. With iPad(s) you have to tap twice to follow a submenu-link.
(also reproducable at the mentioned docs)

Versions:
- iPad (6. gen + 8. gen): 14.4.2
- iPad Air (2. gen): 12.4.8 (+ 12.5.2)
